### PR TITLE
Include SCA policy for Oracle Linux 9 in SPECS

### LINKED ISSUE #2283: SPECS for new SCA Alma Linux 8
Create SPECS as needed for a new OS SCA

PR - https://github.com/wazuh/wazuh/pull/17790 Is ready to merge
Branch - https://github.com/wazuh/wazuh/tree/create-sca-policy-for-alma-linux-os-8

PLEASE confirm with @wazuh/threat-intel before merging.

### LINKED ISSUE #2765: SPECS for new SCA Oracle Linux 9
|Wazuh version|Install type|Action performed|Platform|
|---|---|---|---|
| 4.8.1 | Manager/Agent | All | Oracle Linux 9 |

Since https://github.com/wazuh/wazuh/pull/18741 has been merged is necessary to update SPECS for the new Oracle Linux 9 SCA.

### PR DESCRIPTION
**Pull request by mjcr99**
|Related issue|
|---|
|Closes #2765 |

## Description

This PR adds the necessary modifications to the SPECS to include the new Alma Linux 8 SCA policy.

## Tests

- Build the package in any supported platform
  - [x] Linux
  - [ ] Windows
  - [ ] macOS
  - [ ] Solaris
  - [ ] AIX
  - [ ] HP-UX
- [x] Package installation
- [ ] Package upgrade
- [ ] Package downgrade
- [ ] Package remove
- [ ] Package install/remove/install
- [ ] Change added to CHANGELOG.md

- Tests for Linux RPM
  - [x] Build the package for x86_64
  - [ ] Build the package for i386
  - [ ] Build the package for armhf
  - [ ] Build the package for aarch64
  - [ ] `%files` section is correctly updated if necessary
- Tests for Linux deb
  - [x] Build the package for x86_64
  - [ ] Build the package for i386
  - [ ] Build the package for armhf
  - [ ] Build the package for aarch64
  - [ ] Package install/remove/install
  - [ ] Package install/purge/install
  - [ ] Check file permissions after installing the package
- Tests for macOS
  - [ ] Test the package from macOS Sierra to Mojave
- Tests for Solaris
  - [ ] Test the package on Solaris 10
  - [ ] Test the package on Solaris 11
  - [ ] Check file permissions on Solaris 11 template
- Tests for IBM AIX
  - [ ] `%files` section is correctly updated if necessary
  - [ ] Check the changes from IBM AIX 5 to 7

<details><summary>Manager</summary>

**Manager package build:** https://ci.wazuh.info/job/Packages_builder/180311

**Package installation:**
```
[root@localhost vagrant]# yum install -y https://packages-dev.wazuh.com/warehouse/test/4.8/rpm/var/wazuh-manager-4.8.1-ol9.x86_64.rpm
Oracle Linux 9 BaseOS Latest  (x86_64)                                                                  2.6 MB/s |  18 MB     00:07    
Oracle Linux 9 Application Stream Packages (x86_64)                                                      27 MB/s |  27 MB     00:00    
Oracle Linux 9 UEK Release 7 (x86_64)                                                                    26 MB/s |  25 MB     00:00    
Last metadata expiration check: 0:00:04 ago on Thu 11 Jan 2024 04:56:37 PM UTC.
wazuh-manager-4.8.1-ol9.x86_64.rpm                                                                       19 MB/s | 351 MB     00:18    
Dependencies resolved.
========================================================================================================================================
 Package                            Architecture                Version                         Repository                         Size
========================================================================================================================================
Installing:
 wazuh-manager                      x86_64                      4.8.1-ol9                       @commandline                      351 M

Transaction Summary
========================================================================================================================================
Install  1 Package

Total size: 351 M
Installed size: 856 M
Downloading Packages:
Running transaction check
Transaction check succeeded.
Running transaction test
Transaction test succeeded.
Running transaction
  Preparing        :                                                                                                                1/1 
  Running scriptlet: wazuh-manager-4.8.1-ol9.x86_64                                                                                 1/1 
  Installing       : wazuh-manager-4.8.1-ol9.x86_64                                                                                 1/1 
  Running scriptlet: wazuh-manager-4.8.1-ol9.x86_64                                                                                 1/1 
  Verifying        : wazuh-manager-4.8.1-ol9.x86_64                                                                                 1/1 

Installed:
  wazuh-manager-4.8.1-ol9.x86_64                                                                                                        

Complete!
```

**SCA file found:**
```
[root@localhost vagrant]# ls /var/ossec/ruleset/sca
cis_amazon_linux_1.yml.disabled     cis_debian8.yml.disabled              cis_solaris11.yml.disabled
cis_amazon_linux_2023.yml.disabled  cis_debian9.yml.disabled              cis_sqlserver_2012.yml.disabled
cis_amazon_linux_2.yml.disabled     cis_iis_10.yml.disabled               cis_sqlserver_2014.yml.disabled
cis_apache_24.yml.disabled          cis_mongodb_36.yml.disabled           cis_sqlserver_2016.yml.disabled
cis_apple_macOS_10.11.yml.disabled  cis_mysql5-6_community.yml.disabled   cis_sqlserver_2017.yml.disabled
cis_apple_macOS_10.12.yml.disabled  cis_mysql5-6_enterprise.yml.disabled  cis_sqlserver_2019.yml.disabled
cis_apple_macOS_10.13.yml.disabled  cis_nginx_1.yml.disabled              cis_ubuntu14-04.yml.disabled
cis_apple_macOS_10.14.yml.disabled  cis_oracle_database_19c.yml.disabled  cis_ubuntu16-04.yml.disabled
cis_apple_macOS_10.15.yml.disabled  cis_oracle_linux_9.yml                cis_ubuntu18-04.yml.disabled
cis_apple_macOS_11.1.yml.disabled   cis_postgre-sql-13.yml.disabled       cis_ubuntu20-04.yml.disabled
cis_apple_macOS_12.0.yml.disabled   cis_rhel5_linux.yml.disabled          cis_ubuntu22-04.yml.disabled
cis_apple_macOS_13.x.yml.disabled   cis_rhel6_linux.yml.disabled          cis_win10_enterprise.yml.disabled
cis_apple_macOS_14.0.yml.disabled   cis_rhel7_linux.yml.disabled          cis_win11_enterprise.yml.disabled
cis_centos6_linux.yml.disabled      cis_rhel8_linux.yml.disabled          cis_win2012r2.yml.disabled
cis_centos7_linux.yml.disabled      cis_rhel9_linux.yml.disabled          cis_win2016.yml.disabled
cis_centos8_linux.yml.disabled      cis_rocky_linux_8.yml.disabled        cis_win2019.yml.disabled
cis_debian10.yml.disabled           cis_sles11_linux.yml.disabled         cis_win2022.yml.disabled
cis_debian11.yml.disabled           cis_sles12_linux.yml.disabled         sca_unix_audit.yml.disabled
cis_debian12.yml.disabled           cis_sles15_linux.yml.disabled         web_vulnerabilities.yml.disabled
cis_debian7.yml.disabled            cis_solaris11.4.yml.disabled

```
</details>

<details><summary>Agent</summary>

**Agent package build:** https://ci.wazuh.info/job/Packages_builder/180342

**Package installation:**
```
[root@localhost vagrant]# yum install -y https://packages-dev.wazuh.com/warehouse/test/4.8/rpm/var/wazuh-agent-4.8.1-ol9.x86_64.rpm
Oracle Linux 9 BaseOS Latest  (x86_64)                                                                   23 MB/s |  18 MB     00:00    
Oracle Linux 9 Application Stream Packages (x86_64)                                                      28 MB/s |  27 MB     00:00    
Oracle Linux 9 UEK Release 7 (x86_64)                                                                    24 MB/s |  25 MB     00:01    
Last metadata expiration check: 0:00:04 ago on Thu 11 Jan 2024 05:13:32 PM UTC.
wazuh-agent-4.8.1-ol9.x86_64.rpm                                                                        3.7 MB/s | 9.3 MB     00:02    
Dependencies resolved.
========================================================================================================================================
 Package                          Architecture                Version                           Repository                         Size
========================================================================================================================================
Installing:
 wazuh-agent                      x86_64                      4.8.1-ol9                         @commandline                      9.3 M

Transaction Summary
========================================================================================================================================
Install  1 Package

Total size: 9.3 M
Installed size: 27 M
Downloading Packages:
Running transaction check
Transaction check succeeded.
Running transaction test
Transaction test succeeded.
Running transaction
  Preparing        :                                                                                                                1/1 
  Running scriptlet: wazuh-agent-4.8.1-ol9.x86_64                                                                                   1/1 
  Installing       : wazuh-agent-4.8.1-ol9.x86_64                                                                                   1/1 
  Running scriptlet: wazuh-agent-4.8.1-ol9.x86_64                                                                                   1/1 
  Verifying        : wazuh-agent-4.8.1-ol9.x86_64                                                                                   1/1 

Installed:
  wazuh-agent-4.8.1-ol9.x86_64                                                                                                          

Complete!


```

**SCA file found:**
```
[root@localhost vagrant]# ls /var/ossec/ruleset/sca
cis_oracle_linux_9.yml
```
</details>

### CODE REVIEW

**Comment by ncvicchi** [comment]
LGTM!

**Review by ncvicchi — DISMISSED** [review]

File: debs/SPECS/wazuh-manager/debian/rules [hunk 1]
```diff
@@ -138,6 +138,7 @@ override_dh_install:
 	mkdir -p ${TARGET_DIR}$(INSTALLATION_SCRIPTS_DIR)/sca/debian/10
 	mkdir -p ${TARGET_DIR}$(INSTALLATION_SCRIPTS_DIR)/sca/debian/11
 	mkdir -p ${TARGET_DIR}$(INSTALLATION_SCRIPTS_DIR)/sca/debian/12
+	mkdir -p ${TARGET_DIR}$(INSTALLATION_SCRIPTS_DIR)/sca/ol/9
```
Comment (lchico) [inline debs/SPECS/wazuh-manager/debian/rules:141]: Could you alphabetically order this and correct the [previous one](https://github.com/wazuh/wazuh-packages/pull/2767/files#diff-84ebdeae3dd7c922738d30f63cd384cc8ac00b326d793f6325397b1b02019df1R119), please?

**Review by lchico — CHANGES_REQUESTED** [review]
Tested and works as expected! Thanks, @mjcr99 
I left a minor comment.

File: debs/SPECS/wazuh-manager/debian/rules (hunk 1 above)
Comment (mjcr99) [reply debs/SPECS/wazuh-manager/debian/rules:141]: You are right @lchico but we should do it in several files, I think this task should be addressed in a different issue.

**Review by lchico — APPROVED** [review]
LGTM!

**Review by jotacarma90 — APPROVED** [review]
LGTM

**Review by ncvicchi — APPROVED** [review]

**Pull request merged**